### PR TITLE
Adding doc for script pre-requisite

### PR DESCRIPTION
**Pull request by chiaramapellimt**

### CODE REVIEW

File: docs/guides/setting-up-a-new-consumer.md [hunk 1]
```diff
@@ -14,7 +14,14 @@ per environment.
 
 ## Create a client certificate
 
-1. Run the [generate-client-certificate.sh](/scripts/client_certificates/generate.sh) script with the name of the environment and client.
+1. As a pre-requisite to create a client certificate by running the script below, please ensure that:
```
Comment (CslBing) [inline docs/guides/setting-up-a-new-consumer.md:17]: List this in Prerequisites section above

**Review by CslBing — DISMISSED** [review]
LGTM

File: docs/guides/setting-up-a-new-consumer.md [hunk 2]
```diff
@@ -12,9 +12,17 @@ per environment.
 - [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
 - [Access to Cloud Platform’s Kubernetes cluster](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/getting-started/kubectl-config.html#installing-kubectl)
 
+As a pre-requisite to create a client certificate by running the script below, please ensure that:
+- Select the account you need.SSO registration scopes
+- You run `aws configure sso` and follow the prompts to populate the SSO session name [anything], the SSO start URL [https://madetech.awsapps.com/start], the region [eu-west-2] and SSO registration scopes [simply press Enter].
```
Comment (CslBing) [inline docs/guides/setting-up-a-new-consumer.md:17]: why is that referring to madetech?

Comment (chiaramapellimt) [reply docs/guides/setting-up-a-new-consumer.md:17]: what would the SSO start url for us be?

**Review by CslBing — APPROVED** [review]

**Pull request merged**